README.md

```diff
@@ -5,7 +5,9 @@
 ![rubocop](https://github.com/googleapis/ruby-spanner-activerecord/workflows/rubocop/badge.svg)
 
 __This adapter only supports GoogleSQL-dialect Cloud Spanner databases. PostgreSQL-dialect
-databases are not supported.__
+databases are not supported. You can use the standard PostgreSQL ActiveRecord adapter in
+[combination with PGAdapter](https://github.com/GoogleCloudPlatform/pgadapter/blob/-/samples/ruby/activerecord)
```

Comment:
This link redirects here: https://github.com/GoogleCloudPlatform/pgadapter/tree/postgresql-dialect/samples/ruby/activerecord 
Shouldn't we be using this link instead?

Reply:
The link is deliberately chosen this way. The `../blob/-/...` part is handled by GitHub, so that it always redirects to the current default branch. That means that if we change the default branch name from `postgresql-dialect` to for example `main` in the future, the link remains valid and continues to direct the user to the default branch.

Reply:
Sounds good!